<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>5</y>
      <width>766</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="TabSegment">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>分割</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string>选择</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="OpenOriginalImageBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>打开待分割图像</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="OriginalImageView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>171</width>
         <height>131</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="SegmentBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>161</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>分割</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>分割方法</string>
       </property>
      </widget>
      <widget class="QComboBox" name="SegmentType">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>185</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="SetSegmentParasInOneImageBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>分割参数设置</string>
       </property>
      </widget>
      <widget class="QComboBox" name="RemoveDuplicateType">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>315</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="RemoveDuplicateBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>161</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>去重</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SetMatchParasInOneImageBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>去重参数设置</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>去重方法</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SetSegmentPatchesQuantityBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>分割图元块设置</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>15</y>
        <width>516</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>分割图元展示</string>
      </property>
      <widget class="QListWidget" name="ImagePatchViewInOneImage">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>491</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="SavePatches2DataBaseBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>491</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>分割结果存入数据库</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="TabSearch">
     <attribute name="title">
      <string>检索</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>选择</string>
      </property>
      <widget class="QPushButton" name="OpensampleImageBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>打开样本图</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="SampleImageView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>171</width>
         <height>171</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="SearchBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>161</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>搜  索</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>匹配方法</string>
       </property>
      </widget>
      <widget class="QComboBox" name="MatchType">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>235</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="SetMatchParasSearchBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>匹配参数设置</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>506</width>
        <height>366</height>
       </rect>
      </property>
      <property name="title">
       <string>检索结果</string>
      </property>
      <widget class="QListWidget" name="SuperImagePatchView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>481</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="ImagePatchView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>481</width>
         <height>196</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>批量入库</string>
     </attribute>
     <widget class="QPushButton" name="SetMatchParasAllBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>匹配参数设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetSegmentParasAllBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>分割参数设置</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>701</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>图像库</string>
      </property>
      <widget class="QPushButton" name="OpenImageLibBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>图片库</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ImageLibPath">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="AddinLibBtn">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>20</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>加入图像库</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="SegmentTypeAll">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>96</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="MatchTypeAll">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>96</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>155</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>分割方法</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>匹配方法</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>超图元</string>
     </attribute>
     <widget class="QListWidget" name="AllSuperImagePatchViewInPage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>10</y>
        <width>641</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ShowAllSuperImagePatchesBtn">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>45</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>显示图元</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PreviousPageBtn">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NextPageBtn">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>315</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="log">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>465</y>
      <width>766</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>日志打印</string>
    </property>
    <widget class="QTextEdit" name="LogDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>706</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>log</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
